### PR TITLE
PLATSUP-1024 Update beta/partner program pages in platform docs

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/PLATSUP-1024

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1710268962053919) and give us a `:review-pr:` emoji.

File: docs/_manage/integration-changes/essential-tips-iq.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-title: Rollout cadence and integration quality health tips
+title: Essential Tips for Integrating Quality Health Practices
```
Comment (marinahand) [inline docs/_manage/integration-changes/essential-tips-iq.md:2]: Developer Style guide recommends sentence case - Essential tips for integrating quality health practices

File: docs/_publish/partner-program/partner-program.md [hunk 2]
```diff
@@ -9,32 +9,32 @@ redirect_from:
 
 # Partner Program 
 
-> **The Partner Program is only eligible for public integrations.**
+> **The Partner Program is only available to public integrations.**
 
 The [Zapier Partner Program](https://zapier.com/developer-platform/partner-program) is a program for Zapier’s [6,000+ integration partners](https://zapier.com/apps). It is designed to give all partners a clear path to success for their integrations and reward them with benefits along the way.
 
-As you get more users and maintain a healthy integration, your integration accesses more benefits across our tiered system. We have 4 levels:
+As your integration gains more active users, and maintains a healthy health score, your integration will tier up in our Partner Program. Zapier's Partner Program consistes for 4 tiers:
```
Comment (marinahand) [inline docs/_publish/partner-program/partner-program.md:16]: healthy health score sounds strange 😊 - should that be high health score? satisfactory health score?

File: docs/_publish/partner-program/partner-program.md [hunk 3]
```diff
@@ -9,32 +9,32 @@ redirect_from:
...
 
 
 * Bronze
 * Silver
 * Gold
 * Platinum
 
-Learn more about what [benefits are available in each level](https://zapier.com/developer-platform/partner-program).
+Learn more about what [benefits are available in each tier](https://zapier.com/developer-platform/partner-program).
```
Comment (marinahand) [inline docs/_publish/partner-program/partner-program.md:24]: Learn more about **the** benefits...

File: docs/_publish/partner-program/partner-program.md (hunk 2 above)
Comment (cameronjp) [reply docs/_publish/partner-program/partner-program.md:16]: Haha yeah, it does sound weird. But that is technically (?) the designation in dev platform and how we refer to it in tickets with partners. eg. https://cdn.zappy.app/18ccd475d668beec8da9dd1526a6d3ff.png

Open to your suggestions! I just want the verbiage to match what's shown in dev platform

**Review by marinahand — APPROVED** [review]

**Pull request merged**